### ISSUE TITLE
Internationalization of accessibility 

### ISSUE THREAD

**Issue by Bubblesphere** [clappr/clappr-core#15]
The `string.js` plugin is missing labels used for accessibility.

For example, I would like to be able to dynamically change the aria-label of the fullscreen button based on the current language:
English:
`<button aria-label="fullscreen"></button>`
French:
`<button aria-label="plein écran"></button>`

**Comment by sapolio** [comment]
For exactly the same case, I upgraded my media-control plugin extension.
I rewritten the plugin template by adding an Aria attribute object in two languages (en, ru).
Of course, it would be ideal if attributes such as aria-label and title already had their translations in the strings plugin, and they would already show the hotkeys assigned to them.

**Comment by joaopaulovieira** [comment]
As @sapolio described it very well, the ideal approach would be to have these translations for all elements of media control in the string plugin.

It would be necessary a PR here in the clappr-core to create these translations and another in the clappr-plugins project using these translations in the templates of each element.

Contributions are welcome (;
